tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/queryfixer/entity/TokenImpl.java

```diff
@@ -13,8 +13,11 @@ public TokenImpl (Token token) {
     this.token = token;
   }
 
-  @Override public int getKind() {
-    return token.kind;
+  @Override public Kind getKind() {
+    if (token.kind == 786 || token.kind == 788) {
```

Comment:
I forgot maybe you explained last time why it is 786?

Reply:
786 is the identifier with backticks, but 788 is an identifier without any enclosing.

Reply:
They will be deprecated very soon. Probably next a few PRs when the ZetaSQL is used.